tests/cmd-list-filter-match-exact.tesh

```diff
@@ -15,3 +15,34 @@ $ zk list -q --debug-style -Me --match '["न", "म", "स्", "ते"]'
 >
 >  - Given the Hindi word "नमस्ते":
 >
+
+# Mutliple match flags.
+$ zk list -q --debug-style -Me --match "thread" --match "mut"
+><title>Concurrency in Rust</title> <path>g7qa.md</path> (9 minutes ago)
```

Comment:
The timestamps here were giving me some grief, I ran `make tesh-update` which I guess is the idea? It was pretty violent though so I only included the "new" tests but I assume the timestamps need to be manually changed or ...

Reply:
I think this date issue might have been because I had a `notebook.db` in tree, removing it fixes it as the index is always fresh.

Reply:
Yeah that's right, the database is rebuilt from scratch for every tesh file.